### PR TITLE
chore: Use require instead of import to make worker-interface methods sync

### PR DESCRIPTION
**Pull request by bergundy**
While this could be considered a step in the wrong direction, it simplifies our code a bit.
It will pay off when we get to context reuse.

Tested this manually with the ESM sample, it works.

Broke the experimental `internal` interceptors `dispose` API, it's no longer async.

### CODE REVIEW

File: packages/workflow/src/worker-interface.ts [hunk 1]
```diff
@@ -103,14 +103,12 @@ export async function initRuntime(options: WorkflowCreateOptionsWithSourceMap):
 
   // TODO(bergundy): check if we can use `require` with the ESM sample and make all of the runtime methods sync.
 
-  // @ts-expect-error this is a webpack alias to payloadConverterPath
```
Comment (mjameswh) [inline packages/workflow/src/worker-interface.ts:106]: The prose in this comment still seems useful...

**Review by mjameswh — APPROVED** [review]
Why would that be the wrong direction? I see no problem with require in this context.

**Comment by bergundy** [comment]
Looks like I need to fix a few things here.
I'll look into it.

File: packages/workflow/src/worker-interface.ts (hunk 1 above)
Comment (bergundy) [reply packages/workflow/src/worker-interface.ts:106]: Thanks, I'll add it back.

File: packages/workflow/src/worker-interface.ts [hunk 2]
```diff
@@ -103,14 +103,12 @@ export async function initRuntime(options: WorkflowCreateOptionsWithSourceMap):
 
   // TODO(bergundy): check if we can use `require` with the ESM sample and make all of the runtime methods sync.
```
[the remark this answers is not in the document]
Comment (bergundy) [reply packages/workflow/src/worker-interface.ts:104]: Nope, thanks for catching this

**Pull request merged**